### ISSUE TITLE
Error: Cannot find module 'node_modules/@testing-library/vue/compatibility-scripts/vue-apollo-patch.js'

### ISSUE THREAD

**Issue by jellymann** [testing-library/vue-testing-library#201]
**Describe the bug**

Getting the following error when installing the package:

```
Error: Cannot find module 'node_modules/@testing-library/vue/compatibility-scripts/vue-apollo-patch.js'
```

**To Reproduce**

```
mkdir vue-testing-library-test
cd vue-testing-library-test
yarn add @testing-library/vue@next
```

**Expected behavior**

Package installs with no errors.

**Related information:**

- `@testing-library/vue` version: 6.3.3
- `Vue` version: N/A
- `node` version: v14.4.0
- `yarn` version: 1.22.10

**Comment by afontcu** [comment]
Hi! Thanks for the report. I believe it is due this line: https://github.com/testing-library/vue-testing-library/blob/next/package.json#L15

changing it from `postinstall` to another sensible key (we only need this file when developing and before running tests) would fix it. I'll look into it today.

**Comment by afontcu** [comment]
Okay, it was simple. Got it fixed on [6.3.4](https://github.com/testing-library/vue-testing-library/releases/tag/v6.3.4) 😄  updating VTL will work as expected.

Thank you!
